scripts/dev/dev-rebuild.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+PAYARA_DIR=/usr/local/payara5
+ASADMIN=$PAYARA_DIR/glassfish/bin/asadmin
+DB_NAME=dvndb
+DB_USER=dvnapp
+
+echo "Checking if there is a war file to undeploy..."
+LIST_APP=$($ASADMIN list-applications -t)
+OLD_WAR=$(echo $LIST_APP | awk '{print $1}')
+NEW_WAR=target/dataverse*.war
+if [ ! -z $OLD_WAR ]; then
+  $ASADMIN undeploy $OLD_WAR
+fi
+
+echo "Stopping app server..."
+$ASADMIN stop-domain
+
+echo "Deleting \"generated\" directory..."
+rm -rf $PAYARA_DIR/glassfish/domains/domain1/generated
+
+echo "Deleting ALL DATA FILES uploaded to Dataverse..." 
+rm -rf $PAYARA_DIR/glassfish/domains/domain1/files
+
+echo "Terminating database settings so we can drop the database..."
```

Comment:
just because I'm curious, what does this do?

Reply:
I'm assuming you mean the pg_terminate_backend stuff. It's just a SQL statement I found online somewhere. I didn't dig into it. In practice, it's sometimes necessary to terminate all sessions because you've got a session or two left over from pgAdmin or a terminal. In my experience, you can drop a database with open sessions.

Reply:
Ah, makes sense. I was confused by "Terminating database settings". I think that's a typo and should be "sessions".

Reply:
Oh, a typo. I meant sessions, not settings. Fixed in d3778f360.